src/Core/Program.cs

```diff
@@ -374,6 +375,8 @@ public TypeLibraryDeserializer CreateTypeLibraryDeserializer()
         /// </summary>
         public Dictionary<string, Dictionary<FunctionType, IntrinsicProcedure>> Intrinsics { get; private set; }
 
+        public Dictionary<string, (string, string?, ExternalProcedure)> ExternalProcedures { get; private set; }
```

Comment:
Please add a doc comment for this property.

Reply:
Done in 82b8276